public/content/documentation/web/page-level/form.md

```diff
@@ -1,80 +1,598 @@
+   - Skip-links: Not commonly used for form entry, but useful for jumping to a form landmark.
+   - Tab: Focus moves to each form control in logical order, strong visual focus indicator.
+   - Forms mode: Enabled when screen reader lands on a form field.
 
 2. Test mobile screenreader gestures
-
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: Focus moves to form controls inside the form.
+   - Singletap: Selects a form control.
+   - Doubletap: Activates controls as expected.
+   - Forms mode: Automatically enabled.
 
 3. Listen to screenreader output on all devices
+   - Role: Each input and group is correctly identified (e.g. textbox, combobox, radiogroup).
+   - Name: Every input has a programmatic label (visible label, aria-label, or aria-labelledby).
+   - Group: Groups of related fields are identified with fieldset/legend or aria attributes.
+   - Instructions: Presented before the form or inline, announced in order.
+   - Errors: Errors are announced when inputs are invalid.
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - Group: It typically contains the name and primary navigation of the website
-
-Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/page-level/form](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/page-level/form)
+Full information: [https://www.magentaa11y.com/checklist-web/form/](https://www.magentaa11y.com/checklist-web/form/)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a web form for accessibility
 
-GIVEN THAT I am on a page with a header landmark
+GIVEN THAT I am on a page with a web form
 
 1. Keyboard for mobile & desktop
-
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to move focus to a form field
+      - I SEE focus is visually indicated
+      - I SEE the focus moves in a logical order
 
 2. Desktop screenreader
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver)
+   - AND I use the tab key or arrow keys to move through form fields
+      - I HEAR the role and label for each field
+      - I HEAR instructions associated with form fields
+      - I HEAR error messages when a field is invalid
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+3. Mobile screenreader
+   - WHEN I use a mobile screenreader (TalkBack, VoiceOver)
+   - AND I swipe through the form fields
+      - I HEAR each field's name, role, and state
+      - I HEAR grouped information where applicable
+      - I HEAR errors or hints when applicable
+
+Full information: [https://www.magentaa11y.com/checklist-web/form/](https://www.magentaa11y.com/checklist-web/form/)
+
+## Code examples
+
+### Use semantic HTML
+
+<form aria-label="Sign in">
+  <fieldset>
```

Comment:
I like your idea of changing the order of the code snippet/example but for now -- since all the other criteria do not do this, can we change it back? Can you add an agenda item to https://app.asana.com/1/4329459156007/project/1206165789580137/list/1206165799872638 about this?